### PR TITLE
fixes an ancient viro exploit

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
regenerative coma and superficial healing had a weird threshold inherited from a parent that changed their effects at 6 speed, which has been removed

## Why It's Good For The Game
obviously not intended, given regenerative coma has a stage speed 8 threshold that lowers power back to 1.5, and superficial healing has a stage speed 8 threshold that does the same thing as the hidden one, with more stat reqs. also, i added the latter, and didnt intend for this

## Testing Photographs and Procedure
image

## Changelog
:cl:
fix: fixes an exploit involving superficial healing and regen coma
balance: superficial healing and regen coma no longer have an undocumented stage speed six threshold that doubles healing
/:cl:

### CODE REVIEW

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 1]
```diff
@@ -18,8 +18,6 @@
 /datum/symptom/heal/Start(datum/disease/advance/A)
 	if(!..())
 		return FALSE
-	if(A.stage_rate >= 6) //stronger healing
-		power = 2
 	return TRUE //For super calls of subclasses
```
Comment (PowerfulBacon) [inline code/datums/diseases/advance/symptoms/heal.dm:20]: Threshold descriptions are now incorrect. You may be fixing an exploit, but you are simultanously fixing something completely intended.

Comment (zeskorion) [reply code/datums/diseases/advance/symptoms/heal.dm:20]: alright, ill remove the threshold description from the unused healing symptom, then

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Update PR body and changelog to reflect balance changes.

**Comment by zeskorion** [comment]
> Update PR body and changelog to reflect balance changes.

done

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**